virtual-organizations/EIC.yaml

```
@@ -1,6 +1,10 @@
 AppDescription: EIC collaboration.
 CertificateOnly: false
-Community: The EIC collaboration. 
+Community: The EIC collaboration.
```

```suggestion
Community: The EIC collaboration. 
```

Just to make the diff cleaner